browser/src/Services/Learning/Tutorial/Tutorials/TargetsVimPluginTutorial.tsx

```diff
@@ -0,0 +1,142 @@
+                new Stages.FadeInLineStage(null, 2, Line2b),
+                new Stages.SetBufferStage([Line2a, Line2b, Line2c]),
+            ),
+            new Stages.SetCursorPositionStage(0, 7),
+            new Stages.MoveToGoalStage("Use 'cin(' to move inside the paranthesis", 0, 40),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.SetCursorPositionStage(1, 7),
+            new Stages.MoveToGoalStage("Use 'can(' to move around the paranthesis", 1, 39),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.MoveToGoalStage(
+                "Use 'cil(' to move back inside the last paranthesis",
+                0,
+                40,
+            ),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            Stages.combine(
+                null,
+                new Stages.FadeInLineStage(null, 1, Line3a),
+                new Stages.FadeInLineStage(null, 2, Line3b),
+                new Stages.SetBufferStage([Line3a, Line3b, Line3c]),
+            ),
+            new Stages.SetCursorPositionStage(0, 0),
+            new Stages.MoveToGoalStage("Use 'cin'' to move inside the single quotes", 0, 19),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.SetCursorPositionStage(1, 7),
+            new Stages.MoveToGoalStage(
+                "Use 'cIn\"' to move to first character inside double quotes",
+                1,
+                48,
+            ),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.SetCursorPositionStage(2, 7),
+            new Stages.MoveToGoalStage("Use 'cAn\"' to select around the double quotes", 2, 26),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            Stages.combine(
+                null,
+                new Stages.FadeInLineStage(null, 1, Line4a),
+                new Stages.FadeInLineStage(null, 2, Line4b),
+                new Stages.SetBufferStage([Line4a, Line4b, Line4c]),
+            ),
+            new Stages.SetCursorPositionStage(0, 7),
+            new Stages.MoveToGoalStage("Use 'din,' to delete the next item in the list", 0, 40),
+            new Stages.MoveToGoalStage(
+                "Use 'din_' to delete from the middle of the variable",
+                2,
+                41,
+            ),
+            Stages.combine(
+                null,
+                new Stages.FadeInLineStage(null, 1, Line5a),
+                new Stages.FadeInLineStage(null, 2, Line5b),
+                new Stages.SetBufferStage([Line5a, Line5b, Line5c]),
+            ),
+            new Stages.SetCursorPositionStage(0, 0),
+            new Stages.MoveToGoalStage("Use 'dina' to delete the next programming argument", 1, 31),
```

Comment:
I noticed after the delete it moved quickly to the next stage (so I wasn't quite able to see the impact of my change) - is this what you referred to about adding a pause? I think that might be helpful, just to give the user some feedback about what the delete operation did.